script.js

```diff
@@ -0,0 +1,235 @@
+            alert(textPrompt);
+            //відмальовуємо країни
+            setTimeout(() => setСountries(arrCounty, budget), 1500);
+            addClassByClass("circle", "active")
+            let wordsAstronaut = "Привіт! Клікни на країну в яку б ти хотів поїхати, а я перевірю чи тобі вистачить грошей."
+            changeText(wordsAstronaut, "words-astronaut")
+        }
+    };
+}
+
+
+//функція додає клас по className
+function addClassByClass(classNameElem, classNameAdd) {
+    if (classNameElem) {
+        let elem = document.querySelectorAll(`.${classNameElem}`);
+        elem.forEach(el => {
+            el.classList.add(`${classNameAdd}`);
+        })
+    }
+}
+
+//функція видаляє клас по className
+function removeClassByClass(classNameElem, classNameRemove) {
+    if (classNameElem) {
+        let elem = document.querySelectorAll(`.${classNameElem}`);
+        elem.forEach(el => {
+            el.classList.remove(`${classNameRemove}`);
+        })
+    }
+}
+//функція додає  клас по id
+function addClassById(id, className) {
+    if (id) {
+        let elem = document.getElementById(`${id}`);
+        elem.classList.add(`${className}`);
+    }
+}
+//функція сторює елемент-країну
+function createElemCountry(content, idElem, left, top, budget) {
+    let elem = document.createElement(`div`);
+    elem.innerHTML = `${content}`;
+    let parent = document.getElementById("blok-img");
+    elem.setAttribute("class", "country active-country");
+    elem.setAttribute("id", `${idElem}`);
+    elem.style.left = `${left}`;
+    elem.style.top = `${top}`;
+    elem.addEventListener("click", function (e) {
+        removeClassByClass("country-ok", "country-ok");
+        removeClassByClass("country-not", "country-not")
+        var idElem = e.target.id;;
+        let checkSummRes = checkSumm(budget, arrCounty, idElem)
+        if (checkSummRes.res) {
+            this.setAttribute("class", "country active-country country-ok");
+            setTimeout(() => getNewPrice(checkSummRes.selectPrice, true), 1500);    
+        } else {
+            if (checkSummRes.arr.length > 0) {
+                checkSummRes.arr.forEach(function (elem) {
+                    addClassById(elem, "country-ok")
+                });
+            }
+            this.setAttribute("class", "country active-country country-not");
+        }
+
+    });
+    parent.appendChild(elem);
+};
+//функція сторює елементи країни з масиву
+function setСountries(arr, budget) {
+    arr.forEach(function (elem) {
+        createElemCountry(elem.name, elem.id, elem.stylesLeft, elem.stylesTop, budget)
+    });
+};
+
+//функція сторює елементи країни з масиву
+function changeText(text, id) {
+    if (id) {
+        let elem = document.getElementById(`${id}`);
+        elem.innerHTML = `${text}`;
+    }
+};
+//перевіряємо бюджет
+function checkSumm(summ, arr, id) {
+    let arrNormCounry = [];
+    for (let i = 0; i < arr.length; i++) {
+        if (id == arr[i].id) {

```

Comment:
можно в одному if все перевірити
```suggestion
        const { id: countryId, price: countryPrice } = arr[i]
        if (id == countryId && +summ >= countryPrice) {

```